config_editor.html

```diff
@@ -24,9 +27,52 @@
     <meta name="theme-color" content="#ffffff">
 </head>
 <body>
+    <nav class="navbar navbar-expand-lg navbar-dark fixed-top">
```

Comment:
This doesn't have a mobile navigation bar making it impossible to go anywhere else but 'back' on mobile

Reply:
Resolved with grey background for mobile users.